paper-input-container.html

```diff
@@ -594,12 +610,21 @@
           if (label) {
             this.$.labelAndInputContainer.style.position = 'relative';
           }
+          if (invalid) {
+            cls += ' is-invalid';
+          }
         }
       } else {
         if (_inputHasContent) {
           cls += ' label-is-hidden';
         }
+        if (invalid) {
```

Comment:
@notwaldorf just checking to see if you have any other thoughts on this. Not sure how to/if I need to alter this to make sure the validation state is managed appropriately.